src/framework/global/thirdparty/haw_logger/logger/logdefdest.cpp

```diff
@@ -71,5 +73,8 @@ std::string ConsoleLogDest::name() const
 
 void ConsoleLogDest::write(const LogMsg& logMsg)
 {
-    std::clog << m_layout.output(logMsg) << std::endl;
+    std::string stemp = m_layout.output(logMsg);
+    std::wstring temp = std::wstring(stemp.begin(), stemp.end());
+    OutputDebugString(temp.c_str());
+    OutputDebugString(L"\n");
```

Comment:
This solution can only work for Windows, it won't work for Linux and Macos, it won't even compile

Reply:
@igorkorsukov Is there a specific reason that we use std::clog? It seems to send all output to stderr instead of stdout, and I'm not sure if that's desirable.

Reply:
There is no special reason, in fact it may be better to replace it with `std::cout`

Reply:
Okay, I will do that too

Reply:
@igorkorsukov No, it doesn't work. I had to use the `OutputDebugString()` function. Also, this also  gets displayed along with the regular logs when you are debugging. These were getting printed before this commit:
![image](https://user-images.githubusercontent.com/88136054/177759882-6ea5798b-ebb9-4409-a6df-7c58b165cddd.png)